packages/ckeditor5-dev-release-tools/lib/tasks/creategithubrelease.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { Octokit } = require( '@octokit/rest' );
+
+/**
+ * Create a Github release.
+ *
+ * @param {Object} options
+ * @param {String} options.token Token used to authenticate with GitHub.
+ * @param {String} options.version Name of tag connected with the release.
+ * @param {String} options.repositoryOwner Owner of the repository.
+ * @param {String} options.repositoryName Repository name.
+ * @param {String} options.description Description of the release.
+ * @param {Boolean} options.isPrerelease Indicates whether the release is a pre-release.
```

Comment:
I am not sure about this flag. Perhaps it could be read automatically from the specified version. TBD in the future, before merging the feature branch.